core/server/apps/private-blogging/lib/middleware.js

```diff
@@ -70,7 +70,7 @@ privateBlogging = {
         // NOTE: Redirect to /private if the session does not exist.
         privateBlogging.authenticatePrivateSession(req, res, function onSessionVerified() {
             // CASE: RSS is disabled for private blogging e.g. they create overhead
-            if (req.path.lastIndexOf('/rss/', 0) === 0 || req.path.lastIndexOf('/rss/') === req.url.length - 5) {
+            if (req.path.match(/\/rss\/?/)) {
```

Comment:
I think we need the $ at the end, otherwise it could match e.g. `/rss-post/`.

Reply:
Hmm, it passed all tests :). What are possibilities of RSS path? are they /rss and /tag1/tag2/rss? I think I need to add some tests too.